server/routes/api/recommendations/recommendationsRoutes.ts

```diff
@@ -119,6 +135,18 @@ recommendationsRouter.get(
       }
       const { parent: parentId, babysitter: babysitterId } = req.params;
 
+      const parentValidation = await handler.parentValidation(Number(parentId));
```

Comment:
Here you call a validation function in the handler and that validation function calls the other 2 validations in the hadnler.
So the route only calls 1